lang/1031.json

```diff
@@ -235,5 +235,18 @@
         "i64d_torrents": "%1 Torrent(s)",
         "dl_s_ul_s": "DL: %1/s, UL: %2/s",
         "encrypt_config_file": "Konfigurationsdatei verschlüsseln"
+		"amp_details_panel": "&Detailansicht",
+        "s_of_s": "%1 (von %2)",
+        "downloaded_size": "Heruntergeladen",
+        "amp_status_bar": "&Statusleiste",
+        "amp_filter": "&Filter",
+        "amp_none": "&Keine",
+        "filter_downloading_active": "Downloads (aktiv),
```

Comment:
You're missing an ending quote here (`"`) which breaks the JSON. Add it and I'll merge it!